### PR TITLE
Seeding -

### PR DESCRIPTION
**Pull request by emmanuelmota**
Seeds the DB.

### CODE REVIEW

File: seeding/seeding.js [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+  const decision = randomInt(1, 100, true);
+  if (decision < 50) {
+    return true;
+  }
+  return false;
+};
+// console.log('Super Host Bool: ', superHostBool());
+
+// Rating Generator - This has to be a float
+const rating = () => randomInt(1, 5, true);
```
Comment (chrsbell) [inline seeding/seeding.js:41]: Generate a rating with 1 or 2 decimal places instead of an int

File: seeding/seeding.js [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+// console.log('Random Rating: ', rating());
+
+// Review Count
+const reviewCount = () => randomInt(1, 300, true);
+// console.log('Review Count: ', reviewCount());
+
+// This will output an Array photo OBJs 10 of them for now.
+const imageSeeder = () => {
+  const url = 'www.aws.com/imageurl.jpg';
```
Comment (chrsbell) [inline seeding/seeding.js]: Generate a random url instead of using static url

File: seeding/seeding.js [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+  }
+  return result;
+};
+
+// Seed the DB
+const dataToSeed = () => {
+  let results = [];
+  let count = 1;
+
+  for (let i = 0; i < 100; i++) {
```
Comment (chrsbell) [inline seeding/seeding.js]: Can refactor to use 2 lines (let count = 0) and use count instead of i. Or set roomId to i+1

File: server/index.js [hunk 4]
```diff
@@ -0,0 +1,18 @@
+const express = require('express');
+const { Stay, db } = require('../database/index');
+
+const app = express();
+const port = 3000;
+
+app.get('/stays', (req, res) => {
+  Stay.find({}, (err, data) => {
+    if (err) {
+      res.status(400).send('Error Find All Failed.');
```
Comment (chrsbell) [inline server/index.js]: May be able to send 'err' instead of hard coded error

**Review by chrsbell — CHANGES_REQUESTED** [review]
Requires a few changes

File: seeding/seeding.js (hunk 3 above)
Comment (emmanuelmota) [reply seeding/seeding.js]: Fixed

File: seeding/seeding.js (hunk 1 above)
Comment (emmanuelmota) [reply seeding/seeding.js:41]: Fixed

File: seeding/seeding.js (hunk 2 above)
Comment (emmanuelmota) [reply seeding/seeding.js]: Fixed.

File: server/index.js (hunk 4 above)
Comment (emmanuelmota) [reply server/index.js]: Fixed

**Pull request merged**